sns/parity_math.c

```diff
@@ -668,18 +720,23 @@ M0_INTERNAL int m0_parity_math_init(struct m0_parity_math *math,
 	return M0_RC(ret);
 }
 
-static void xor_calculate(struct m0_parity_math *math,
-			  const struct m0_buf *data,
-			  struct m0_buf *parity)
+static int xor_calculate(struct m0_parity_math *math,
+			 const struct m0_buf *data,
+			 struct m0_buf *parity)
 {
-        uint32_t          ei; /* block element index. */
-        uint32_t          ui; /* unit index. */
-        uint32_t          block_size = data[0].b_nob;
+	uint32_t	  ei; /* block element index. */
+	uint32_t	  ui; /* unit index. */
+	uint32_t	  block_size = data[0].b_nob;
```

Comment:
Remove tabs between variable type and variable name.

Reply:
Done.